sdk/core/amqp-common/src/errors.ts

```diff
@@ -518,8 +519,24 @@ export function isSystemError(err: any): boolean {
     err.code &&
     typeof err.code === "string" &&
     (err.syscall && typeof err.syscall === "string") &&
-    (err.errno &&
-      (typeof err.errno === "string" || typeof err.errno === "number"))
+    (err.errno && (typeof err.errno === "string" || typeof err.errno === "number"))
+  ) {
+    result = true;
+  }
+  return result;
+}
+
+/**
+ * Utility to check if given object is an instance of a browser based WebSocket error
```

Comment:
Until we foresee a need to use this externally, I would suggest not to `export` this utility function